.circleci/config.yml

```diff
@@ -5,12 +5,11 @@ jobs:
       tag: "3.8"
     steps:
       - checkout
-      - python/load-cache
-      - python/install-deps
-      - python/save-cache
+      - python/install-packages:
+          pkg-manager: poetry
       - run:
           command: |
-            python -m pytest --cov nway --cov-report xml
+            poetry run python -m pytest --cov nway --cov-report xml
```

Comment:
Since I'm not familiar with poetry, why does this need to be run with `poetry`? Does `poetry` create its own virtual environment or something? Would it interact with our conda environments?

Reply:
yes, poetry creates its own virtualenv. We would not need conda environments (except perhaps to easily install poetry itself).
https://python-poetry.org/docs/basic-usage/

Reply:
I guess my concern is for repos like `ophys_etl_pipelines` where we need to install the repo alongside Suite2P (in a conda environment). Is it possible for poetry to install into existing environments?